Package.swift

```diff
@@ -9,6 +9,12 @@ func hasEnvironmentVariable(_ name: String) -> Bool {
   return ProcessInfo.processInfo.environment[name] != nil
 }
 
+/// Use the `NonDarwinLogger` even if `os_log` can be imported.
+///
+/// This is useful when running tests using `swift test` because xctest will not display the output from `os_log` on the
+/// command line.
+let forceNonDarwinLogger = hasEnvironmentVariable("SOURCEKITLSP_FORCE_NON_DARWIN_LOGGER")
```

Comment:
Won't this mean build + test + install require 3 builds instead of 1?

Reply:
Since none of the dependencies need to be rebuilt, the recompile when flipping the default is fairly fast (20s on my machine) and I think that’s quite acceptable to get log output when running tests.